### ISSUE TITLE
dist version for Android, SDK 6.0+

### ISSUE THREAD

**Issue by ssx** [Sitata/titanium-google-analytics#62]
Would someone be able to release an Android 'dist' version compiled for use with SDK6.0.0+?

**Comment by aurusapps** [comment]
@ssx see the fork by alan89 https://github.com/alan89/titanium-google-analytics/commit/39ac2c777b0cdbd55f45c3f176144d87fe035a98

**Comment by caiocost** [comment]
When i use this dist i got "Application stopped running" with no error on log

**Comment by anthonychung** [comment]
Hi, I've made a distribution version for reference. 

https://github.com/anthonychung/titanium-google-analytics/tree/develop/android/dist

It may not be appropriate for your set up. It was made for the combination scenario when someone has to use it with titanium-google-analytics with ti.goosh push notifications. 

I've created a pull request for convenience for historical reference for the diff between the branches. 
Pull request is likely to be rejected since most people won't need the push notification GCM combo.
https://github.com/Sitata/titanium-google-analytics/pull/63

All the best,
Anthony
